specification/ai/HealthInsights/HealthInsights.RadiologyInsights/route.radiologyinsights.tsp

```diff
@@ -13,24 +12,15 @@ using TypeSpec.Http;
 namespace AzureHealthInsights;
 
 interface RadiologyInsights {
-  #suppress "@azure-tools/typespec-azure-core/no-rpc-path-params" "Service uses a jobId in the path"
   @summary("Get Radiology Insights job details")
   @tag("RadiologyInsights")
   @doc("Gets the status and details of the Radiology Insights job.")
   @get
-  @route("/radiology-insights/jobs/{id}")
-  getJob is HealthInsightsLongRunningPollOperation<RadiologyInsightsResult>;
+  getJob is HealthInsightsOperations.ResourceRead<RadiologyInsightsJob>;
 
-  #suppress "@azure-tools/typespec-azure-core/long-running-polling-operation-required" "Polling through operation-location"
-  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "There is no long-running RPC template in Azure.Core"
   @summary("Create Radiology Insights job")
   @tag("RadiologyInsights")
   @doc("Creates a Radiology Insights job with the given request body.")
   @pollingOperation(RadiologyInsights.getJob)
-  @route("/radiology-insights/jobs")
-  createJob is HealthInsightsLongRunningRpcOperation<
-    RadiologyInsightsData,
-    RadiologyInsightsResult,
-    RadiologyInsightsInferenceResult
-  >;
+  createJob is HealthInsightsOperations.LongRunningResourceCreateOrReplace<RadiologyInsightsJob>;
```

Comment:
If the `LongRunningResourceCreateOrReplace` is from Azure.Core, the template variable should be the `Result` model, i.e., `RadiologyInsightsInferenceResult`

Reply:
Not sure if the PUT is a mistake https://github.com/Azure/azure-rest-api-specs/pull/27614/files#r1493516639

Reply:
@asaflevi-ms 

From your description this PUT seems to be the new design here (the PUT to status monitor)
https://microsoft-my.sharepoint.com/:w:/p/mikekistler/EVNINn3afyhOljAju4XTBi0BdEuImmmZH7w0Cs-KvisdWg?e=eHR08H

But note that there is still difference, e.g. "kind" property is required in that design (though I can see why it is not there -- currently the only "kind" is createJob).

Also, there maybe things that missing in codegen/SDK, as this design not yet get to the guideline doc. So, current codegen/SDK would likely still follow [this LRO PUT to resource](https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#put-with-additional-long-running-processing) (and the template `Azure.Core.LongRunningResourceCreateOrReplace` also describes this PUT) , which is different from the one above.